libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -1,9 +1,14 @@
+#ifndef MQTT_LIGHTWEIGHT_H
+#define MQTT_LIGHTWEIGHT_H
+
 #include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>
 
 #include "config.h"
 
+#define MQTT_PACKET_TYPE_CONNECT         ( ( uint8_t ) 0x10U )
```

Comment:
Is this needed in header file or can be moved to implementation file?

Reply:
It's needed here.  The application may match `type == MQTT_PACKET_TYPE_CONNECT`.